### ISSUE TITLE
US:  ARIA has been defined on the incorrect element

### ISSUE THREAD

**Issue by Equis17** [qlik-oss/sn-table#872]
The aria-pressed attribute is incorrectly defined on the elements. This should be reserved for toggle buttons only.

Tasks:
- [Crl/buttons in header  ] #814
